source/concepts.tex

```diff
@@ -53,7 +53,7 @@
 input values for which the expression is required to be well-defined.
 
 \pnum
-Expressions required by this document to be equality-preserving are further
+Expressions required by the library Clauses to be equality-preserving are further
```

Comment:
This sentence really doesn't need a qualifier at all: "Expressions required to be equality-preserving are further required to..." seems just fine.

Reply:
Fixed.